doc/examples/MainClass.rst

```diff
@@ -39,6 +39,16 @@ Get organization by name
     >>> org.login
     u'PyGithub'
 
+Get enterprise consumed licenses by name
+------------------------
+
+.. code-block:: python
+
+    >>> enterprise = g.get_enterprise_consumed_licenses("PyGithub")
+    >>> enterprise_consumed_licenses = enterprise.get_enterprise_consumed_licenses()
+    >>> enterprise_consumed_licenses.total_seats_consumed
+    u'5000'
```

Comment:
why there is a `u` prefix?

Are you using python2?

Reply:
No, I use python3. I just followed others in MainClass.rst such like u'Wan Liuyang' in Get user by name section.

Reply:
I think we should remove this `u` prefix, we don't support python2 anymore.

Reply:
Sure. Will remove it. May I remove others in the file as well?

Reply:
maybe leave others as-is, not related to this PR.

Reply:
I just removed it only this PR's one.